jekyll-assets/_layouts/boxes404.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    {% include head.html %}
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/boxes.css">
+  </head>
+  <body>
+    {% include fonts.html %}
+    {% include header.html %}
+    {% include toptitle404.html %}
+    {% include subtitle404.html %}
```

Comment:
These two lines are the only difference between this file and `jekyll-assets/_layouts/boxes.html`. So I _could_ have done something clever with an `{% if .... %}` to prevent potential duplication, but this probably suffices for now :smile:

Reply:
I had a quick fiddle, and this approach was easier than I was expecting, so that's what I've now done! :grinning: